demo/elastic/wazuh_cf_kibana.sh

```diff
@@ -297,6 +297,18 @@ install_plugin(){
 
 }
 
+
+optimize_kibana(){
+  systemctl stop kibana
+  echo "Optimizing app" >> /tmp/deploy.log
+  cd /usr/share/kibana
+  NODE_OPTIONS="--max-old-space-size=2048" ./bin/kibana --optimize --allow-root
```

Comment:
Why doing the optimize as root?

Reply:
Fixed on 4672cd5e65ea9ed2db6de6ed06472afe74eddcca.